client/src/store/actions/authActions.tsx

```diff
@@ -30,4 +24,25 @@ export const loginUser = (
   }
 };
 
-export const authStartLoading = () => ({ type: TYPES.LOGIN_LOADING });
+export const registerUser = (
+  registerData: RegisterData
+): ThunkAction<void, {}, {}, RegisterAction> => async dispatch => {
+  dispatch({
+    type: TYPES.REGISTER_LOADING
+  });
+
+  axios
+    .post("/api/users/register", registerData)
+    .then(resp => {
```

Comment:
We can destructure this value

`.then({ data } => {`

Reply:
Done